packages/react-server/core/components/RootElement.js

```diff
@@ -23,7 +23,17 @@ class RootElement extends React.Component {
 
 				// Okay, now we've complained about it
 				// sufficiently, let's go ahead and update.
-				this.props.childProps = childProps;
+
+				// Because this.props.childProps is writable/configurable:false
+				// we update the entirety this.props through a clone
+				var propstemp = {};
+				for (var k in this.props) {
```

Comment:
Should probably either iterate over `Object.keys(this.props)` here or add a `this.hasOwnProperty(k)` check.